decidim-meetings/app/cells/decidim/meetings/highlighted_meetings_for_component_cell.rb

```diff
@@ -44,6 +44,15 @@ def past_meetings_count
       def upcoming_meetings_count
         @upcoming_meetings_count ||= meetings.upcoming.count
       end
+
+      def cache_hash
+        hash = []
+        hash << "decidim/meetings/highlighted_meetings_for_component"
+        hash << meetings.cache_key_with_version
+        hash.push(current_user.try(:id))
```

Comment:
Is there anything user-specific for the highlighted meetings? At least looking from the UI, I cannot identify any.

Reply:
In the method `def meetings` from this cell, only the meetings visible for the current user are fetch, that's why

Reply:
Ah makes sense, thanks!